### PR TITLE
add some metrics and set up otlp exporting

### PR DESCRIPTION
**Pull request by whyrusleeping**

### CODE REVIEW

File: cmd/bigsky/main.go [hunk 1]
```diff
@@ -64,6 +66,9 @@ func run(args []string) {
 		&cli.BoolFlag{
 			Name: "jaeger",
 		},
+		&cli.BoolFlag{
+			Name: "otlp-export",
```
Comment (Jacob2161) [inline cmd/bigsky/main.go:70]:
```suggestion
			Name: "otlp-export",
			EnvVars: []string{"OTLP_EXPORT"},
```

File: cmd/bigsky/main.go [hunk 2]
```diff
@@ -110,9 +115,15 @@ func run(args []string) {
 		&cli.StringFlag{
 			Name: "disk-blob-store",
 		},
+		&cli.StringFlag{
+			Name:    "env-type",
```
Comment (Jacob2161) [inline cmd/bigsky/main.go:119]:
```suggestion
			Name:    "env-type",
			Usage: "environment type (test, staging, production)",
```

**Review by Jacob2161 — CHANGES_REQUESTED** [review]
I've already added the receiver part for datadog-agent.

With this change (I added one env var)  we should be able to deploy with this for e.g. production:

```bash
BGS_ENVIRONMENT=production
OTEL_EXPORTER_OTLP_ENDPOINT=http://localhost:4318
OTLP_EXPORT=true
```

**Comment by whyrusleeping** [comment]
this needs to be rebased and updated to use the latest opentrace metrics package

**Comment by whyrusleeping** [comment]
this was merged in a different PR

**Pull request closed without merging**